pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/MethodReturnsInternalArrayRule.java

```diff
@@ -6,9 +6,9 @@
 
 import java.util.List;
 
+import net.sourceforge.pmd.lang.java.ast.ASTAnyTypeDeclaration;
```

Comment:
@KroArtem This is the Checkstyle violation. This import should be grouped with other `net.sourceforge.pmd` imports below, in alphabetical order

Reply:
Thanks! Those violation was referring to an xml that was (I suppose) generated during clean verify phase and then deleted, that's why I was wondering what exactly is wrong.

Reply:
FYI for the future - you should see a similar output as on travis when running the build locally: 
https://travis-ci.org/pmd/pmd/jobs/539209393#L3475

```
[INFO] --- maven-checkstyle-plugin:3.0.0:check (checkstyle-check) @ pmd-java ---
[INFO] There is 1 error reported by Checkstyle 8.18 with /net/sourceforge/pmd/pmd-checkstyle-config.xml ruleset.
[ERROR] src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/MethodReturnsInternalArrayRule.java:[10] (imports) ImportOrder: Wrong order for 'org.jaxen.JaxenException' import.
```
It appears a couple of lines above the summary message...

Reply:
Thanks! The thing is that those ruleset does not exist in repository that's why firstly I was a bit frustrated.
`[INFO] There is 1 error reported by Checkstyle 8.18 with /net/sourceforge/pmd/pmd-checkstyle-config.xml ruleset.`